test/CMakeLists.txt

```diff
@@ -1,9 +1,13 @@
 file(GLOB_RECURSE sources "*.cpp")
-list(REMOVE_ITEM sources "${CMAKE_CURRENT_SOURCE_DIR}/fuzzer.cpp")
+file(GLOB_RECURSE toolsources "tools/*.cpp")
+list(REMOVE_ITEM sources ${toolsources})
 file(GLOB_RECURSE headers "*.h")
 
 add_executable(soltest ${sources} ${headers})
 target_link_libraries(soltest PRIVATE libsolc solidity lll evmasm devcore ${Boost_UNIT_TEST_FRAMEWORK_LIBRARIES})
 
-add_executable(solfuzzer fuzzer.cpp)
+add_executable(solfuzzer tools/fuzzer.cpp)
 target_link_libraries(solfuzzer PRIVATE libsolc evmasm ${Boost_PROGRAM_OPTIONS_LIBRARIES})
+
+add_executable(syntaxTestTool tools/syntaxTestTool.cpp TestHelper.cpp libsolidity/SyntaxTest.cpp libsolidity/AnalysisFramework.cpp)
```

Comment:
This should later be expanded to semantic tests, too. What about `isoltest' (interactive soltest)?